examples/example-tls/src/main/java/io/grpc/examples/helloworldtls/HelloWorldClientS2A.java

```diff
@@ -0,0 +1,118 @@
+/*
+ * Copyright 2020 The gRPC Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.grpc.examples.helloworldtls;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+import io.grpc.ManagedChannel;
+import io.grpc.StatusRuntimeException;
+import io.grpc.examples.helloworld.GreeterGrpc;
+import io.grpc.examples.helloworld.HelloReply;
+import io.grpc.examples.helloworld.HelloRequest;
+import io.grpc.netty.GrpcSslContexts;
+import io.grpc.netty.NettyChannelBuilder;
+import io.netty.handler.ssl.SslContext;
+import io.netty.handler.ssl.SslContextBuilder;
+import java.io.File;
+import java.util.concurrent.TimeUnit;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import javax.net.ssl.SSLException;
+import javax.annotation.Nonnull;
+
+// TODO: change flags to use com.google.common.flags
+
+/**
+ * Establishes a TLS 1.3 connection with a greeter service.
+ *
+ * args:
+ * [ServerAddress] In the form of host:port
+ * [clientCertChainFilePath] File Path to Client Certificate
+ * [clientPrivateKeyFilePath] File Path to Client Key
+ * [trustCertCollectionFilePath] File Path to CA Certificate
+ */
+public class HelloWorldClientS2A {
+    private static final Logger logger = Logger.getLogger(HelloWorldClientS2A.class.getName());
+    private final ManagedChannel channel;
+    private GreeterGrpc.GreeterBlockingStub blockingStub;
+
+    private static SslContext buildSslContext(String clientCertChainFilePath,
+                                              String clientPrivateKeyFilePath,
+                                              String trustCertCollectionFilePath) throws SSLException {
+      checkNotNull(clientCertChainFilePath, "clientCertChainFilePath");
```

Comment:
Please make the error messages in `checkNotNull` a bit more verbose, e.g. "clientCertChainFilePath should not be null."